hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/service/utils/BinaryGasEstimator.java

```
@@ -51,10 +51,17 @@ public long search(
             contractCallContext.reset();
 
             long mid = (hi + lo) / 2;
-            HederaEvmTransactionProcessingResult transactionResult = call.apply(mid);
+            HederaEvmTransactionProcessingResult transactionResult = null;
+
+            try {
+                transactionResult = call.apply(mid);
+            } catch (Exception ignore) {
```

Sonar is probably going to complain for the empty catch clause. Anyway, it would probably be better to have at least a comment within the block with some reasoning behind the empty block.

---

Reverted this change, because will think of a prettier way to fix that